$NetBSD: patch-ab,v 1.2 2014/12/04 04:28:36 mef Exp $

Use wish from tk, instead of wishx (added by mef) 

--- pclock.sh	2002/05/09 12:07:01	1.1
+++ pclock.sh	2002/05/09 12:07:17
@@ -33,7 +33,7 @@
 # PCLOCK_HOME=${PCLOCK_HOME:-/home/david/src/projclock}
 
 if [ -f $THISDIR/projclock.tcl ] ; then
-	exec wishx $THISDIR/projclock.tcl
+	exec wish $THISDIR/projclock.tcl
 else
-	exec wishx $PCLOCK_HOME/projclock.tcl
+	exec wish $PCLOCK_HOME/projclock.tcl
 fi
